candies/cli/parsers/standard.py

```diff
@@ -62,7 +60,32 @@ def parse(self, args: Iterable[str]) -> Iterable[Invocation]:
         return invocations
 
 
-def configure(parser: ArgumentParser, command: Command, prefix: str = '.'):
+def help(command: Command) -> Type:
+    """Constructs a subclass of an `argparse.HelpFormatter` for a `command`."""
+
+    class Help(HelpFormatter):
+        """Defines a format of the `--help` message."""
+
+        def format_help(self):
+            if command.help_ is not None:
+                return command.help_(command)
+            else:
+                return super(Help, self).format_help()
+
+    return Help
+
+
+def new_parser(*args, **kwargs) -> ArgumentParser:
+    """Creates a new parser."""
+    return ArgumentParser(*args, **kwargs)
+
+
+def new_subparser(subparsers: Any) -> Callable[[Any], ArgumentParser]:
+    """Returns a function to create a new subparser."""
+    return subparsers.add_parser
+
+
+def configured(new: Callable, command: Command, prefix: str = '.'):
```

Comment:
And here as well.

Reply:
Where? There is `Callable[[Any], ArgumentParser]`. Or you mean something else?

Reply:
Fixed.